src/EmployeeUtils.java

```diff
@@ -0,0 +1,50 @@
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.Scanner;
+
+public class EmployeeUtils {
+
+    public static int lineCalculator() throws FileNotFoundException {
```

Comment:
nazwy metody to poiwnny byc czasowniki. POiwnny okreslac czynnosc ktora ta metoda wykonuje. Tu np. 'countLines`. Dodalbym, tez  nazwe pliku jako argument.

Reply:
OK, zmienione.